nixos/modules/services/video/frigate.nix

```diff
@@ -30,6 +30,18 @@ let
 
   filteredConfig = converge (filterAttrsRecursive (_: v: !elem v [ null ])) cfg.settings;
 
+  configFile =
+    pkgs.runCommand "frigate-config"
+      {
+        preferLocalBuilds = true;
+      }
+      ''
+        cp ${format.generate "frigate.yml" filteredConfig} $out
+        export CONFIG_FILE=$out
+        export PYTHONPATH=${cfg.package.pythonPath}
+        ${cfg.package.python.interpreter} -m frigate --validate-config
```

Comment:
If we [cannot](https://github.com/NixOS/nixpkgs/issues/429985#issuecomment-3141727276) use a "buildtime" frigate[1], maybe this check should be conditional on non-cross setups?

[1] `cfg.package` is "runtime" frigate, I think "buildtime" would be `pkgs.buildPackages.frigate`